### PR TITLE
feat(zbctl): allows scope to be supplied via command line parameter

### PR DESCRIPTION
**Pull request by jessesimpson36**
example usage:

```
./zbctl --address zeebe-gateway.dev.jlscode.com:443 \
    --clientId zeebe \
    --clientSecret EasySecret \
    --authzUrl 'https://dev.jlscode.com/auth/realms/camunda-platform/protocol/openid-connect/token' \
    --audience zeebe-api --scope camunda-identity status
```

## Description

This allows users to set the scope for zbctl. For some reason, this scope attribute appears mandatory in 8.5 when installed with identity. I'm not sure why. 

## Related issues

closes #

### CODE REVIEW

**Review by npepinpe — CHANGES_REQUESTED** [review]
I would suggest making the help message consistent with the other options. Other than that, looks good :)

That said, I assume just setting the env var now without the flag would work anyway, no? :shrug:

**Comment by jessesimpson36** [comment]
I think you're right, the environment variable would work.  However, I started writing this PR before I knew the environment variable existed.  by that point, might as well follow through.

File: clients/go/cmd/zbctl/internal/commands/root.go [hunk 1]
```diff
@@ -78,6 +79,7 @@ func Execute() {
 }
 
 func init() {
+	rootCmd.PersistentFlags().StringVar(&scopeFlag, "scope", "", fmt.Sprintf("Specify the scope"))
```
[the remark this answers is not in the document]
Comment (jessesimpson36) [reply clients/go/cmd/zbctl/internal/commands/root.go]: I do like your message a lot better. I didn't quite know how to describe it.

**Comment by mustafadagher** [comment]
@jessesimpson36 
❌ May I ask you to squash your commits into one with the message that follows our [Contributions Guide](https://github.com/camunda/zeebe/blob/main/CONTRIBUTING.md#commit-message-header)? 
PS: your first commit message was perfect! It's the second one that was generated by accepting github suggestion that's the issue

❌ Also, can you please rebase your PR branch on `main` to pass the `Tasklist merge queue CI`?

Thanks

**Comment by jessesimpson36** [comment]
Sounds good. I rebased  on main, and squashed the 2nd commit since it wasn't necessary to have that commit on it's own.

**Review by npepinpe — APPROVED** [review]
:+1:

**Comment by npepinpe** [comment]
While not a bug fix, I think it's small enough to backport to 8.5, and as you mentioned this is more or less required for users on 8.5.

**Comment by megglos** [comment]
FYI the scope param is actually optional in the default SM setups, the failure on the scope was a bug in the go client, a [fix](https://github.com/camunda/zeebe/pull/18154) is ready

**Pull request merged**